xml/art_sles_rpi_quick.xml

```diff
@@ -51,8 +51,8 @@
   <para>
    To be able to use &productnamearch; on the &rpi;, a 64-bit Arm* compatible
```

Comment:
Shouldn't we be using an entity for Arm here?

Reply:
Let's consider such cleanups as follow-ups once the upcoming bulk of contents is in.

Reply:
I've found a couple more unused entities even.